services/economics-rewards/src/delegators.ts

```diff
@@ -0,0 +1,87 @@
+import { DelegationData, Delegator } from './types';
+import { getContractInstance } from './contract';
+import { saveDelegator } from './database';
+
+const getDelegations = async (
```

Comment:
```suggestion
const getAndPersistDelegations = async (
```

Would be great to have it expressed in the name that the method not only returns them but saves them too.